### PR TITLE
Odh manifest typo fix

### PR DESCRIPTION
**Pull request by goern**

### CODE REVIEW

File: manifests/odh/odh-admins.yaml [hunk 1]
```diff
@@ -3,4 +3,4 @@ kind: Group
 metadata:
   name: odh-admins
 users:
-  - user1
+  - "b64:kube:admin"
```
Comment (mamurak) [inline manifests/odh/odh-admins.yaml:6]: @goern What does this change mean?

File: odh-kfp-modelmesh.md
Comment (mamurak) [inline odh-kfp-modelmesh.md:1]: @goern There are a couple of merge conflicts in this file. Could you please remove them? Thanks.

**Review by mamurak — CHANGES_REQUESTED** [review]
@goern Thanks for preparing this PR! Please refer to the review comments. Thanks.

File: manifests/odh/odh-admins.yaml (hunk 1 above)
Comment (goern) [reply manifests/odh/odh-admins.yaml:6]: this generally enables the kube-admin user to be a odh admin.

**Review by mamurak — APPROVED** [review]

**Review by mamurak — APPROVED** [review]

**Comment by mamurak** [comment]
Merged. Thank you for your contribution, @goern!

**Pull request merged**